packages/office-ui-fabric-react/src/components/Persona/Persona.tsx

```diff
@@ -105,7 +105,8 @@ export class Persona extends BaseComponent<IPersonaProps, {}> {
             className,
             PERSONA_SIZE[size],
             PERSONA_PRESENCE[presence as PersonaPresenceEnum],
-            showSecondaryText && styles.showSecondaryText
+            showSecondaryText && styles.showSecondaryText,
+            coinSize && styles.coinSizeNoPresence
```

Comment:
This logic belongs down in the PersonaCoin. PersonaCoin should take in a prop named "hidePresence", which hides the presence icon if it is set to true (by not including it in DOM at all). That prop is set to true if there is a custom coinsize passed in to Persona.tsx. This ensures a cleaner separation between the components.

Reply:
I just pushed an iteration where the PersonaCoin only renders a Presence if no coinSize is specified. Good call on not including it in the DOM. It's a pretty simple change - but I can iterate adding a "hidePresence" prop if you think it would add more value.